_shared/project/Makefile

```diff
@@ -18,12 +18,14 @@ $(call help,make devdata,load development data and environment variables)
 devdata: python
 	@tox -qe dev --run-command 'python bin/make_devdata'
 {% endif %}
-{% if cookiecutter.get("_directory") == "pyramid-app" %}
+{% if cookiecutter.get("_directory") in ["pyapp", "pyramid-app"] %}
 
 .PHONY: dev
 $(call help,make dev,run the whole app \(all workers\))
 dev: python
 	@pyenv exec tox -qe dev
+{% endif %}
+{% if cookiecutter.get("_directory") == "pyramid-app" %}
```

Comment:
`make web` actually applies to the `pyramid-app` cookiecutter only, not `pyapp` or `pypackage`.